utils/hsfs_utils.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+import json
+import hsfs
+
+from typing import Dict, Any
+from pydoop import hdfs
+from pyspark.sql import SparkSession
```

Comment:
I believe these imports will lead to problems on environments without pyspark and pydoop installed. Haven't tried yet though.

Reply:
ah sorry didn't see it's outside the python directory

Reply:
ah sorry didn't see the file is outside the python package